src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -547,6 +593,46 @@ class BasicSettings extends React.Component {
             </FieldRow>
           </React.Fragment>
         }
+
+        {/* Advanced CPU Topology Options */}
+        <ExpandCollapse id={`${idPrefix}-advanced-options`} textCollapsed={msg.advancedCpuTopologyOptions()} textExpanded={msg.advancedCpuTopologyOptions()}>
```

Comment:
Try it like:
```jsx
<Row>
  <Col offset={3} cols={7}>
    <ExpandCollapse ...>
      <Grid ...>
        <FieldRow>
          ...
        </FieldRow>
        ...
      </Grid>
    </ExpandCollapse>
  </Col>
</Row>
```

The outer-most `offset={3} cols={7}` will push the collapse section over to line up with the input fields.

The inner `FieldRow` components will still have 3 cols for the label and 7 cols for the input fields, just that the total space is now within the outer `Col`'s 7 columns.

Further, you could change `FieldRow` to make the component columns adjustable and set it to 4 or 5 if you want them to take less horizontal space.

Reply:
https://github.com/oVirt/ovirt-web-ui/issues/1264#issuecomment-696258357 So I asked the UX about the expected look again and it seems that I will have to find a way how to shorten the CPU Topology drop downs but to leave the rest of the design as it is.

Reply:
Ok, 2 things.

1. The advanced section is below the regular field Grid, so it can be its own Grid.

2. You can use the `Col` to control width in a generic way and not have to tweak the css like you did.  That being said, straight CSS manipulation is more reasonable in other circumstances.

My suggestion (after adding `fieldCols` as a prop that defaults to 7 to the `FieldRow` component):
```jsx
    <Form ...>
      <Grid className={style['settings-container']}>
        ... the standard fields ...
      </Grid>

      <ExpandCollapse ...>
        <Grid className={style['settings-container']}>
          <FieldRow fieldCols={3} label={msg.virtualSockets()} id={`${idPrefix}-topology-sockets`}>
           <SelectBox
              ... 
            />
          </FieldRow>

          ...more advanced field rows...
        </Grid>
      </ExpandCollapse>
    </Form>
```

Reply:
> The advanced section is below the regular field Grid, so it can be its own Grid.

I don't think it is below the regular field Grid, but I can move it below, however it seems that it doesn't help to achieve the expected look (other than straight CSS manipulation).